game_eggs/minecraft/proxy/java/velocity/velocity.toml

```diff
@@ -72,25 +72,25 @@ enable-player-address-logging = true
 [servers]
 # Configure your servers here. Each key represents the server's name, and the value
 # represents the IP address of the server to connect to.
-lobby = "127.0.0.1:30066"
-factions = "127.0.0.1:30067"
-minigames = "127.0.0.1:30068"
+lobby = "172.18.0.1:30066"
```

Comment:
This will break the panel updating the address on its own.

Reply:
Reverted to 127.0.0.1